src/js/containers/PluginSpecificCacheCard/PluginSpecificCacheCard.js

```diff
@@ -39,17 +44,10 @@ class PluginSpecificCacheCard extends Component {
                             <p><FormattedMessage id="container.pluginSpecificCacheCard.description" /></p>
                         </CardContent>
                         <CardControl>
-
-                            { (this.props.cacheCardValue === false) ? (
-                                <Button type="primary" onClick={this.handleModalOpen.bind(this)}>
-                                    <FormattedMessage id="container.pluginSpecificCacheCard.button.enable" />
-                                </Button> 
-                                ) :
-                                <Button onClick={this.handleChange.bind(this, false)}>
-                                    <FormattedMessage id="container.pluginSpecificCacheCard.button.disable" />
-                                </Button> 
-                            }
-
+                            <Toggle label=""
```

Comment:
Rahul and I talked more about it when I built the PluginSpecificCacheTagCard and decided the Enable/Disable button was more confusing than a toggle.